README.md

```diff
@@ -110,7 +110,7 @@ You can run `rustfmt --help` for more information.
 * [Emacs](https://github.com/rust-lang/rust-mode)
 * [Sublime Text 3](https://packagecontrol.io/packages/BeautifyRust)
 * [Atom](atom.md)
-* Visual Studio Code using [RustyCode](https://github.com/saviorisdead/RustyCode) or [vsc-rustfmt](https://github.com/Connorcpu/vsc-rustfmt)
+* Visual Studio Code using [vscode-rust](https://github.com/editor-rs/vscode-rust) or [vsc-rustfmt](https://github.com/Connorcpu/vsc-rustfmt)
```

Comment:
We're linking to the source here, so I'm not sure how important a marketplace presence is. rls-vscode will be added to the marketplace at some point. Although I think it is fine to link to vscode-rust, I'm not happy endorsing it as the only vscode solution at this point.